cmake/AddInputFileTests.cmake

```diff
@@ -170,4 +179,21 @@ ${Python_EXECUTABLE} ${CMAKE_SOURCE_DIR}/tools/CleanOutput.py -v \
 --input-file $2 --output-dir ${CMAKE_BINARY_DIR}\n"
 )
 
+# Write command to execute an input file and clean its output into a shell
+# script, which makes it easier to chain multiple commands
+file(
+  WRITE
+  ${PROJECT_BINARY_DIR}/tmp/InputFileExecuteCheckH5AndClean.sh
+  "\
+#!/bin/sh\n\
```

Comment:
should this have `-e` added? I don't remember what that's needed to correctly stop the intermediate steps in a script (or if that's a bash-ism)